### DISCUSSION TITLE
7困难图开荒失败 打了三次之后 alas还是想要再打一次 多次提示次数不够后 alas点击过多退出

### DISCUSSION THREAD

**Discussion by jingyill** [LmeSzinc/AzurLaneAutoScript#1262] [Q&A] [answered]
我多半是设置有问题 但是就是不知道该怎么设置困难图开荒 求大佬解答 大佬有时间就看看吧 万分感谢
另外，大佬有设置困难一天打三次的额度吗 还有就是大佬nb

[问题log.txt](https://github.com/LmeSzinc/AzurLaneAutoScript/files/8987279/log.txt)

设置是这样审的
![Snipaste_2022-06-27_02-40-59](https://user-images.githubusercontent.com/40361123/175829469-68100808-acde-4ac0-824d-9bb992635f89.png)
![Snipaste_2022-06-27_02-41-15](https://user-images.githubusercontent.com/40361123/175829478-e80fd90d-0d6e-47f1-b747-e968e409ebee.png)

**Comment by XionghuiLi** [comment, marked as answer]
你的停止条件是通关并且三星；一般情况，困难图只打三次的话是打不了三星的；次数用完了，停止条件没达到，alas肯定会继续出击，但出击无效，多次点击就报错了

**Reply by jingyill** [reply]
感谢感谢 >！其实这几个选项还是弄不太明白，刚刚才用alas

**Reply by jingyill** [reply]
那么大佬 我困难开荒的停止条件设为100%之后，那个前往下一关不停 要勾选吗？不勾选感觉才能正常，不然等心情值冷却完了还是会继续点击，可能alas没有设置困难一天只能打3次。ps：我已经试了一遍还是报错。感谢大佬的解答

**Reply by XionghuiLi** [reply]
1、alas的每日任务有困难图每天打三次的功能，前提是你选的关卡已通关（开荒请关闭）
2、alas的开荒功能主线和困难是放在一起的，困难图开荒你自己给个出击次数的停止条件就好了
3、自信一点

**Reply by jingyill** [reply]
那行 那我每天给个3的出击值就行 谢了大佬

**Comment by XionghuiLi** [comment]
还有就是，开荒为什么要开周会和自律
